function/index.js

```diff
@@ -79,6 +81,16 @@ async function handler(event, context) {
             Object.values(s3Keys)[1]
         );
 
+        // Check if the application has come from within CICA
+        // the channel value will be set to telephone if it does
+        const internalTraffic = s3ApplicationData?.channel === 'telephone';
```

Comment:
Does this `channel` property exclusively derive from the query parameter that is added to the URL of cica-web? Can there be a situation where we want to test internally, but that variable has not been set correctly, therefore that traffic would "fall through"?

There has been chat of determining things using the IP of the office. If it originates from that IP (on site/VPN), then it can be treated as internal. Is the `channel` property additionally updated depending on IP at any point?

Reply:
AFAIK, the `channel` parameter works correctly. It's set when the application is created, then never gets updated. If there are known issues then I guess we'd need to fix those, but the tempus broker currently writes this to tariff, so as far as it's concerned this value should be correct.